scripts/sync.py

```diff
@@ -161,7 +166,15 @@ def do(args, Lochness):
                 lochness.attempt(Module.sync, Lochness, subject, dry=args.dry)
 
     # annonymize PII
-    dpanonymize.lock_lochness(Lochness, pii_table_loc=Lochness['pii_table'])
+
+    # if Lochness['s3_selective_sync']:
+        # dpanonymize.lock_lochness(
+                # Lochness,
+                # pii_table_loc=Lochness['pii_table'],
+                # s3_selective_sync = Lochness['s3_selective_sync'])
+    # else:
+        # dpanonymize.lock_lochness(
+                # Lochness, pii_table_loc=Lochness['pii_table'])
```

Comment:
I think now I understand our mismatch about block comments. They should have `#` at the same vertical indent or enclosed in 

```
'''
some code
'''
```

so they are easy to uncomment in future. For the above setting, you would have to scroll left and right to remove all the `#`s.

Reply:
I know you're using VIM, so for easy comment & uncomment, `nerdcommenter` is awesome.
https://github.com/preservim/nerdcommenter 
Select lines, and just type `<leader>cc` for comment out, `<leader>cu` for removing comments.

Reply:
a77b0ab